user_guide/admin_index.rst

```diff
@@ -1,5 +1,7 @@
-System Management and Administration
-====================================
+.. _user-guide-admin-tools:
+
+Tools for the System Management and Administration
+==================================================
 The section describes tools for system management and administration, aimed to be used primarily by the system 
 administrators.
 Please note that in may cases the tools may be not available to usual users, subject to the permissions defined. 
```

Comment:
edit: Please note that in many cases, the tools may be not available to the usual users, subject to the permissions defined.